### ISSUE TITLE
Simple, single line output when there is a file with incorrect formatting

### ISSUE THREAD

**Issue by nogweii** [google/yamlfmt#161]
So that `yamlfmt`'s output can be parsed by other tools to highlight which files need to be fixed, like [reviewdog](https://github.com/reviewdog/reviewdog). Much like the existing `-quiet` flag, but without the preamble.

I'd imagine taking the existing `-quiet` output:

```
2024/02/28 15:23:20 The following files will have formatting changes:

foo/bar.yaml
```

and turning it into:

```
foo/bar.yaml: [warning] the file is not formatted correctly
```

(The exact wording is not required, just a demonstration.)

**Comment by braydonk** [comment]
Hi @nogweii,

Thanks for the suggestion. It's not the first time someone's asked for something like this, I'll find a way to address it. I think I'll come up with some sort of adapter setup so it's possible to implement other kinds of output formats. Will work on this when I have time over the next week.

**Comment by braydonk** [comment]
This feature is now released in v0.12.0
